src/_reducers/checkedFilters.reducer.js

```diff
@@ -0,0 +1,29 @@
+import constants from '../_constants';
+import initialState from './initialState';
+
+const { FILTER_SELECTED } = constants;
+
+const filtersReducer = (state = initialState.checkedFilters, action) => {
+  switch (action.type) {
+    case FILTER_SELECTED: {
```

Comment:
There's an eslint rule causing a problem once the braces are removed i.e. `Unexpected lexical declaration in case block. (no-case-declarations)` @sachungo